wire-ios-data-model/Source/Model/User/Availability/Availability.swift

```diff
@@ -20,3 +20,18 @@
 public enum Availability: Int, CaseIterable {
     case none, available, busy, away
 }
+
+extension Availability: CustomStringConvertible {
+    public var description: String {
```

Comment:
What is this conformance for? Don't we get this behavior (printing a representative value) free these days in many places?

Reply:
Please ignore this comment. I now see behavior is different when called from a different target.